app-shared/src/androidJvmMain/kotlin/io/github/droidkaigi/confsched/component/OneStepForwardController.kt

```diff
@@ -0,0 +1,42 @@
+package io.github.droidkaigi.confsched.component
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.runtime.snapshotFlow
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import androidx.navigation3.runtime.NavKey
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.map
+
+@Composable
+internal fun rememberOneStepForwardControllerForDesktop(
```

Comment:
@kilalabu 
Following the comments below, I have changed the name to one that clearly indicates it is intended for the JVM.
This should resolve any concerns preventing the review from proceeding.
https://github.com/DroidKaigi/conference-app-2025/pull/554#discussion_r2327052954